### ISSUE TITLE
Add NumberRunoff to a Contest

### ISSUE THREAD

**Issue by jdmgoogle** [usnistgov/Voting#8]
There's a `NumberElected` field in `CandidateContest`, but nothing about how many people could advance to a runoff (e.g., `NumberRunoff`). While it's likely the number is 2, there are some elections where more than two people can quality to advance to a runoff election. It would be nice to be able to support this scenario.

**Comment by JDziurlaj** [comment]
I agree that this scenario should be accommodated. How would you structure this? General speaking that ENR model has not distinguished between nomination and "election" contests.

**Comment by samueldana** [comment]
John, are you saying that a NumberRunoff attribute would only be relevant to 'nomination' contests?
If so, how do nomination contests differ from an 'election' contest? I'm not really familiar with the concept.

For now, I've added NumberRunoff to the CandidateContest type on version2 branch of ElectionResultsReporting
https://github.com/usnistgov/ElectionResultsReporting/tree/version2

**Comment by JDziurlaj** [comment]
If the feed is about a single election, the distinction between nomination and election doesn't need to be made at all (a single contest cannot both nominate and elect someone).

**Comment by jdmgoogle** [comment]
[Louisiana Senate elections](https://en.wikipedia.org/wiki/United_States_Senate_election_in_Louisiana,_2016)

If someone gets above 50% in November, then they win. If no one does, then the top two go to a runoff. No one knows until the end of the election as to which one it is.

**Comment by JDziurlaj** [comment]
Added in ERR V2.0.
